openquake/commonlib/logictree.py

```diff
@@ -1178,7 +1178,8 @@ def apply_branchset(self, branchset_node, branchset):
                 branch.child_branchset = branchset
 
     def _get_source_model(self, source_model_file):
-        return open(os.path.join(self.basepath, source_model_file))
+        return open(os.path.join(self.basepath, source_model_file),
+					encoding='utf-8')
```

Comment:
The indentation is wrong here

Reply:
There are tabs, but I have fixed them after merging.